lib/elixir/test/elixir/calendar/iso_test.exs

```diff
@@ -84,12 +82,31 @@ defmodule Calendar.ISOTest do
         Calendar.ISO.day_of_year(2018, 2, 30)
       end
 
-      assert_raise ArgumentError, "invalid date: 2017-11-00", fn ->
```

Comment:
Please revert. If we are already checking the arguments inside, there is no need to do check it in the function clause. It is just more work without any benefits.

Reply:
done